### PR TITLE
TNT_03_Accent-Color

### PR DESCRIPTION
**Pull request by YasharHND**

### CODE REVIEW

File: src/client/styles/_note-menu-bar.scss [hunk 1]
```diff
@@ -60,6 +60,31 @@
...
+
+      .color-option {
+        background-color: blue;
+        width: 19px;
+        height: 20%;
+        border-radius: 27px;
+        margin: 5px 0;
+
+        &:hover {
+          background-color: darken($note-sidebar-color, 10%);
```
Comment (ghost) [inline src/client/styles/_note-menu-bar.scss:84]: Please add this line here:
  cursor: pointer;

**Review by ghost — CHANGES_REQUESTED** [review]
comment added

**Review by amin10043 — APPROVED** [review]
Review submitted

File: src/client/styles/_note-menu-bar.scss (hunk 1 above)
Comment (YasharHND) [reply src/client/styles/_note-menu-bar.scss:84]: @YasharSandbox Done!

File: src/client/styles/_note-menu-bar.scss [hunk 2]
```diff
@@ -60,6 +60,31 @@
     font-size: 0.7rem;
     font-weight: 400;
   }
+
+  .note-menu-bar-section-colors {
+    position: relative;
+    top: 6px;
+
+    .color-box {
+      position: absolute;
```
Comment (YasharHND) [inline src/client/styles/_note-menu-bar.scss:69]: TEST

**Review by ghost — APPROVED** [review]
Review submitted

**Pull request merged**